server/sonar-ce/src/main/java/org/sonar/ce/queue/PurgeCeActivities.java

```diff
@@ -48,16 +50,27 @@ public PurgeCeActivities(DbClient dbClient, System2 system2) {
   public void start() {
     try (DbSession dbSession = dbClient.openSession(false)) {
       Calendar sixMonthsAgo = Calendar.getInstance();
-      sixMonthsAgo.setTimeInMillis(system2.now());
+      long now = system2.now();
+      sixMonthsAgo.setTimeInMillis(now);
       sixMonthsAgo.add(Calendar.DATE, -180);
+      System.out.println("now in prod=" + now + " -180 days in prod = " + sixMonthsAgo.getTimeInMillis() + " - " + sixMonthsAgo.getTime());
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace this use of System.out or System.err by a logger. ([squid:S106](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS106))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2Fsl%2F7.1%2FSONAR-9218%2Fpurge_scanner_contexts_after_4_weeks&issues=AWGQFmXVFMFpNJFuf3Se&open=AWGQFmXVFMFpNJFuf3Se)

[ISSUE_KEY]: AWGQFmXVFMFpNJFuf3Se